home-service/src/main/java/com/softserveinc/ita/homeproject/service/exceptions/ApiException.java

```diff
@@ -0,0 +1,13 @@
+package com.softserveinc.ita.homeproject.service.exceptions;
+
+import org.springframework.http.HttpStatus;
+
+public class ApiException extends Exception{
```

Comment:
Why do we have `Api...` in service? Why is it checked, do we really need to make it checked?

Reply:
Ok, understood the issue, thanks